lab/RobotWalks/RobotWalks.js

```diff
@@ -0,0 +1,38 @@
+import { WIDTH, HEIGHT } from "../../sketchlib/dimensions.js";
+import { draw_primitive } from "../../sketchlib/p5_helpers/draw_primitive.js";
+import { DirectionalPad } from "../lablib/DirectionalPad.js";
+import { ArcRobot } from "./ArcRobot.js";
+
+const DPAD = new DirectionalPad();
```

Comment:
For mobile, add some virtual touch buttons on the left/right halves of the screen for left/right